### PR TITLE
RadioCard improvements

### PR DESCRIPTION
**Pull request by zuzannanap**

### CODE REVIEW

File: packages/design-system/src/stories/atoms/RadioCard.stories.tsx [hunk 1]
```diff
@@ -37,11 +36,14 @@ export default {
     labelText: {
       description: "string",
     },
-    labelPosition: {
+    radioPosition: {
       control: "select",
       options: ["left", "right"],
       description: "left | right",
     },
```
Comment (monikaracka) [inline packages/design-system/src/stories/atoms/RadioCard.stories.tsx:43]:
```suggestion
    radioPosition: {
      control: "select"
    },
```

File: packages/design-system/src/components/RadioCard/index.tsx [hunk 2]
```diff
@@ -48,10 +49,10 @@ export const RadioCard = forwardRef<HTMLInputElement, RadioCardProps>(
       <label
         htmlFor={id}
         className={classNames(
-          "align-center bg-radioCard-base rounded-radioCard-radiusBase [&:has(:focus-visible)]:ring-radioCard-base-focused flex min-w-fit cursor-pointer gap-2 border p-3.5 [&:has(:focus-visible)]:ring-4",
+          "align-center flex min-w-fit cursor-pointer gap-2 rounded-radioCard-radiusBase border bg-radioCard-base p-3.5 [&:has(:focus-visible)]:ring-4 [&:has(:focus-visible)]:ring-radioCard-base-focused",
           {
-            "flex-row": labelPosition === "left",
-            "flex-row-reverse": labelPosition === "right",
+            "flex-row": radioPosition === "left",
+            "flex-row-reverse": radioPosition === "right",
```
Comment (monikaracka) [inline packages/design-system/src/components/RadioCard/index.tsx:55]: Create styles with object like in other components e.g:
`const positionVariantStyles: GenericKeys = {
left: "flex-row",
right: "flex-row-reverse"
}`

**Review by monikaracka — CHANGES_REQUESTED** [review]
Figma:
Screenshot 2023-06-22 at 13 08 26
Storybook:
Screenshot 2023-06-22 at 13 10 24

⬆️  -styles are different here .

Screenshot 2023-06-22 at 13 17 28
 
⬆️  - we don't want this additional margin here.

File: packages/design-system/src/components/RadioCard/index.tsx [hunk 3]
```diff
@@ -61,17 +62,19 @@ export const RadioCard = forwardRef<HTMLInputElement, RadioCardProps>(
...
-        />
+        {isRadioVisible && (
+          <Radio
+            name={name}
+            id={id}
+            checked={checked}
+            disabled={disabled}
+            onChange={onChange}
+            className={classNames({
+              hidden: size === "sm",
```
[the remark this answers is not in the document]
Comment (zuzannanap) [reply packages/design-system/src/components/RadioCard/index.tsx:73]: it happens while auto-saving document and works well

**Review by monikaracka — APPROVED** [review]

**Pull request merged**